doc/conf.py

```diff
@@ -304,4 +304,5 @@ def setup(app):
     'examples_dirs': examples_dirs,
     'gallery_dirs': gallery_dirs,
     'find_mayavi_figures': find_mayavi_figures,
-    }
+    'expected_failing_examples': set(['../examples/plot_raise.py'])
```

Comment:
I would just put a list here for simplicity (I seem to remember it is turned into a set in the code anyway right?)